client/error/client_error.go

```diff
@@ -0,0 +1,28 @@
+package error
+
+import "github.com/scalar-labs/dl/ledger/status_code"
```

Comment:
I am not 100% sure, it has been a while since I have used Go, but I think Go linters complain about using underscores in package names.  Maybe it is some semi-official convention or something.  Doesn't mean we have to follow it but might be worth checking out the Go lint results.

Reply:
@ypeckstadt 
Thanks.
I checked the golint results manually.
(It didn't automatically show on my VSCode. I guess I didn't configure the plugin correctly)
The underscore package name is not recommended.
I will fix it.

It also has many other lint errors that can be fixed.
I will fix them in another PR.

Reply:
@ypeckstadt 
I have another PR #11 for the lint issue.
(I already merge that PR's branch to this PR)
Please take a look at it.
Thank you.